src/sfizz/Region.cpp

```diff
@@ -381,6 +393,226 @@ bool sfz::Region::parseOpcode(const Opcode& opcode)
+        }
+        break;
+    case hash("fil_random"):
+        {
+            const auto filterIndex { opcode.firstParameter().value_or(1) - 1 };
+            if (!extendIfNecessary(filters, filterIndex + 1))
+                return false;
+
+            setValueFromOpcode(opcode, filters[filterIndex].random, Default::filterRandomRange);
+        }
+        break;
+    case hash("fil_type"):
+        {
+            const auto filterIndex { opcode.firstParameter().value_or(1) - 1 };
+            if (!extendIfNecessary(filters, filterIndex + 1))
+                return false;
+
+            switch (hash(opcode.value)) {
+                case hash("lpf_1p"): filters[filterIndex].type = FilterType::kFilterLpf1p; break;
+                case hash("hpf_1p"): filters[filterIndex].type = FilterType::kFilterHpf1p; break;
+                case hash("lpf_2p"): filters[filterIndex].type = FilterType::kFilterLpf2p; break;
+                case hash("hpf_2p"): filters[filterIndex].type = FilterType::kFilterHpf2p; break;
+                case hash("bpf_2p"): filters[filterIndex].type = FilterType::kFilterBpf2p; break;
+                case hash("brf_2p"): filters[filterIndex].type = FilterType::kFilterBrf2p; break;
+                case hash("bpf_1p"): filters[filterIndex].type = FilterType::kFilterBpf1p; break;
+                case hash("brf_1p"): filters[filterIndex].type = FilterType::kFilterBrf1p; break;
+                case hash("apf_1p"): filters[filterIndex].type = FilterType::kFilterApf1p; break;
+                case hash("lpf_2p_sv"): filters[filterIndex].type = FilterType::kFilterLpf2pSv; break;
+                case hash("hpf_2p_sv"): filters[filterIndex].type = FilterType::kFilterHpf2pSv; break;
+                case hash("bpf_2p_sv"): filters[filterIndex].type = FilterType::kFilterBpf2pSv; break;
+                case hash("brf_2p_sv"): filters[filterIndex].type = FilterType::kFilterBrf2pSv; break;
+                case hash("lpf_4p"): filters[filterIndex].type = FilterType::kFilterLpf4p; break;
+                case hash("hpf_4p"): filters[filterIndex].type = FilterType::kFilterHpf4p; break;
+                case hash("lpf_6p"): filters[filterIndex].type = FilterType::kFilterLpf6p; break;
+                case hash("hpf_6p"): filters[filterIndex].type = FilterType::kFilterHpf6p; break;
+                case hash("pink"): filters[filterIndex].type = FilterType::kFilterPink; break;
+                case hash("lsh"): filters[filterIndex].type = FilterType::kFilterLsh; break;
+                case hash("hsh"): filters[filterIndex].type = FilterType::kFilterHsh; break;
+                case hash("peq"): filters[filterIndex].type = FilterType::kFilterPeq; break;
+                default:
+                     filters[filterIndex].type = FilterType::kFilterNone;
+                    DBG("Unknown filter type: " << std::string(opcode.value));
+            }
+        }
+        break;
+    // Performance parameters: EQ
```

Comment:
I don't see `fil_gain` handled, opcode serves as level parameter to lsh/hsh/peq.